src/Services/privateApiService.js

```diff
@@ -1,21 +1,36 @@
-import axios from 'axios';
+import axios from 'axios'
 
 const config = {
-    headers: {
-        Group: 91                //Aqui va el ID del equipo!!
-    }
+  headers: {
+    Group: 91, //Aqui va el ID del equipo!!
+  },
 }
+const axiosInstance = axios.create({
+  baseURL: 'http://ongapi.alkemy.org/private/api',
+})
 
-const Get = () => {
-    axios.get('https://jsonplaceholder.typicode.com/users', config)
-    .then(res => console.log(res))
-    .catch(err => console.log(err))
+axiosInstance.interceptors.request.use((config) => {
+  config.headers.Authorization = getAuthorizationHeader()
+  config.headers.Group = 91
+  return config
+})
+
+export const getAuthorizationHeader = () => {
+  const token = localStorage.getItem('token')
+  if (!token) return
+  return { Authorization: `Bearer: ${token}` }
 }
 
-export default Get;
+export const privatePut = async (url, id, params = {}) => {
+  const Id = id ? `/${id}` : ''
+  axiosInstance.put(`${url}${Id}`, { params })
```

Comment:
you should return this value, also, you should send directly the params.

Reply:
done! https://github.com/alkemyTech/OT91-server/pull/53/commits/e440d28a88c5ad9bf72de5fbbec4f6ffdf953f24